runtime/lua/vim/lsp.lua

```diff
@@ -26,6 +26,8 @@ local lsp = {
   codelens = require('vim.lsp.codelens'),
   semantic_tokens = semantic_tokens,
   util = util,
+  omnifunc = require('vim.lsp.completion').omnifunc,
```

Comment:
The interface is `vim.lsp.omnifunc`, so this new "completion" module should stay private.
```suggestion
  omnifunc = require('vim.lsp._completion').omnifunc,
```